src/components/Footer/index.jsx

```diff
@@ -0,0 +1,5 @@
+import Container from "./styles";
+
+export const Footer = ({ children }) => {
+  return <Container>{children}</Container>;
```

Comment:
O footer é repetido, por que seu conteúdo é um children, assim o componente fica praticamente inútil e restrito à estilização, bom colocar logo o conteúdo que se repete.